bowler/helpers.py

```diff
@@ -39,6 +39,7 @@ def print_selector_pattern(
             for child in node.children:
                 print_selector_pattern(child, results, filename)
             click.echo("> ", nl=False)
+    click.echo("\n", nl=False)
```

Comment:
Why not just `click.echo()`?

Reply:
No particular reason, I just wanted to keep with the `nl=False` style in the surroundings.
Should I change?